sgx/src/attestation_types/ti.rs

```diff
@@ -21,6 +22,31 @@ pub struct TargetInfo {
     reserved2: [u64; 25],
 }
 
+#[repr(C, align(128))]
+/// Pass information from the source enclave to the target enclave
+pub struct ReportData(pub [u8; 64]);
+
+#[cfg(feature = "asm")]
+impl TargetInfo {
+    /// Generate a report to the specified target with the included data.
+    pub fn get_report(&self, data: &ReportData) -> report::Report {
```

Comment:
```suggestion
    /// Generate a report to the specified target with the included data.
    ///
    /// # Safety
    /// This function is unsafe because it executes an `enclu` instruction which
    /// is only available on processors that support SGX.
    pub unsafe fn get_report(&self, data: &ReportData) -> report::Report {
```